tips/TIP-0025/core-rest-api.yaml

```diff
@@ -2865,13 +2914,35 @@ components:
                 vByteFactoKey:
                   description: Defines the factor to be used for key/lookup generating fields.
                   type: integer
+            baseToken:
+              description: Gives info about the base token the network uses.
+              properties:
+                name:
+                  type: string
+                  description: The name of the base token of the network.
+                tickerSymbol:
+                  type: string
+                  description: Ticker symbol of the token to be displayed on trading platforms.
+                unit:
+                  type: string
+                  description: The primary unit of the token.
+                decimals:
+                  type: integer
+                  description: Number of decimals the primary unit is divisible up to.
+                subunit:
+                  type: string
+                  description: The name of the smallest possible denomination of the primary unit. subunit * 10^decimals = unit
+                "useMetricPrefix":
+                  type: boolean
+                  description: Wheteher to use metric prefixes for disaplying unit.
```

Comment:
Please also add the required fields, since `subunit` is optional

Reply:
fixed in https://github.com/iotaledger/tips/pull/57/commits/b6cdd1e3d16ead8fdbd233b64f37aee0cc4bc151